### PR TITLE
Add `The Followpoint: BTMC, the Face of osu!` news post

### PR DESCRIPTION
**Pull request by 0x84f**
if anyone has issues with the title speak up now or be silent forevermore
[preview](https://osu2.roanh.dev/home/news/2024-03-07-the-followpoint-btmc-the-face-of-osu)

- [x] make sure BTMC is ok with the changes made
- [x] re-time

### CODE REVIEW

File: news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+date: 2024-03-07 10:00:00 +0000
+---
+
+Join us for an interview with BTMC, one of osu!'s most iconic content creators!
+
+![](/wiki/shared/news/2024-03-07-the-followpoint-btmc-the-face-of-osu/banner.png)
+
+[BTMC](https://osu.ppy.sh/users/3171691) is definitely a name in the community that doesn't need any introduction, their influence reaching every corner of osu!'s vast community while still continuously improving and innovating their content, all in the goal of serving the community.
+
+Next to content creation they are also a top osu! player, event organizer, and community contributor with tenures in [Project Loved](/wiki/Community/Project_Loved) and osu! World Cup casting.
```
Comment (Walavouchey) [inline news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md:13]:
```suggestion
Next to content creation they are also a top osu! player, event organiser, and community contributor with tenures in [Project Loved](/wiki/Community/Project_Loved) and osu! World Cup casting.
```

File: news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+*Moving onto "IRL", I believe the Roundtable was the first time you hosted an IRL event, how did you come to the idea of it?*
+
+> The original idea came from Pishi and [Kroytz](https://osu.ppy.sh/users/2339768), and was originally inspired by the events hosted by Beyond the Summit. I was hanging with them, and we realized that technically the only thing we were missing was money, so when I mentioned I'd cover it we got straight to work.
+
+*Hosting such an event is a huge undertaking, both financially and logistically. What were some of the biggest/unexpected obstacles you faced during its organization?*
```
Comment (Walavouchey) [inline news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md:67]:
```suggestion
*Hosting such an event is a huge undertaking, both financially and logistically. What were some of the biggest/unexpected obstacles you faced during its organisation?*
```

File: news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "The Followpoint: BTMC, The Face of osu!"
```
Comment (Walavouchey) [inline news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md:3]: would be like this

```suggestion
title: "The Followpoint: BTMC, the Face of osu!"
```

open to other ideas though

File: news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "The Followpoint: BTMC, the Face of osu!"
```
Comment (iSlodinxOsu) [inline news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md:3]:
```suggestion
title: "The Followpoint: BTMC, the face of osu!"
```

Comment (iSlodinxOsu) [reply news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md:3]: Not sure if uppercase is suppose to be there

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md (hunk 4 above)
Comment (Walavouchey) [reply news/2024/2024-03-07-the-followpoint-btmc-the-face-of-osu.md:3]: it's in title case, so yes

**Review by peppy — APPROVED** [review]
Approving title and general content, haven't done any deep checking but reads sound enough!

**Comment by 0x84f** [comment]
still waiting on BTMC to let me know whether the changes made to their answers are fine

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**